code/datums/wounds/cuts.dm

```diff
@@ -0,0 +1,259 @@
+	sound_effect = 'sound/weapons/slice.ogg'
+	processes = TRUE
+	wound_type = WOUND_TYPE_CUT
+	treatable_by = list(/obj/item/stack/medical/suture, /obj/item/stack/medical/gauze)
+	treatable_by_grabbed = list(/obj/item/gun/energy/laser)
+	treatable_tool = TOOL_CAUTERY
+	treat_priority = TRUE
+
+	/// How much blood we start losing when this wound is first applied
+	var/initial_flow
+	/// When we have less than this amount of flow, either from treatment or clotting, we demote to a lower cut or are healed of the wound
+	var/minimum_flow
+	/// How fast our blood flow will naturally decrease per tick, not only do larger cuts bleed more faster, they clot slower
+	var/clot_rate
+
+	/// Once the blood flow drops below minimum_flow, we demote it to this type of wound. If there's none, we're all better
+	var/demotes_to
+
+	/// How much staunching per type (cautery, suturing, bandaging) you can have before that type is no longer effective for this cut NOT IMPLEMENTED
+	var/max_per_type
+	/// The maximum flow we've had so far
+	var/highest_flow
+	/// How much flow we've already cauterized
+	var/cauterized
+	/// How much flow we've already sutured
+	var/sutured
+
+	/// The current bandage we have for this wound (maybe move bandages to the limb?)
+	var/obj/item/stack/current_bandage
+	/// A bad system I'm using to track the worst scar we earned (since we can demote, we want the biggest our wound has been, not what it was when it was cured (probably moderate))
+	var/datum/scar/highest_scar
+
+/datum/wound/brute/cut/apply_wound(obj/item/bodypart/L, silent = FALSE, datum/wound/brute/cut/old_wound = null)
+	. = ..()
+	blood_flow = initial_flow
+	if(old_wound)
+		blood_flow = max(old_wound.blood_flow, initial_flow)
+		if(old_wound.severity > severity && old_wound.highest_scar)
+			highest_scar = old_wound.highest_scar
+			old_wound.highest_scar = null
+		if(old_wound.current_bandage)
+			current_bandage = old_wound.current_bandage
+			old_wound.current_bandage = null
+
+	if(!highest_scar)
+		highest_scar = new
+		highest_scar.generate(L, src, add_to_scars=FALSE)
+
+/datum/wound/brute/cut/remove_wound(ignore_limb, replaced)
+	if(!replaced && highest_scar)
+		already_scarred = TRUE
+		highest_scar.lazy_attach(limb)
+	return ..()
+
+/datum/wound/brute/cut/get_examine_description(mob/user)
+	if(!current_bandage)
+		return ..()
+
+	var/bandage_condition = ""
+	// how much life we have left in these bandages
+	switch(current_bandage.absorption_capacity)
+		if(0 to 1.25)
+			bandage_condition = "nearly ruined "
+		if(1.25 to 2.75)
+			bandage_condition = "badly worn "
+		if(2.75 to 4)
+			bandage_condition = "slightly bloodied "
+		if(4 to INFINITY)
+			bandage_condition = "clean "
+	return "<B>The cuts on [victim.p_their()] [limb.name] are wrapped with [bandage_condition] [current_bandage.name]!</B>"
+
+/datum/wound/brute/cut/receive_damage(wounding_type, wounding_dmg, wound_bonus)
+	if(wounding_type == WOUND_SHARP)
+		blood_flow += 0.05 * wounding_dmg
```

Comment:
would it be possible to add a way for cuts to self-upgrade if you manage to get blood_flow high enough to reach the next "tier"?

Reply:
I'm electing not to right now since that'd make it a bit too easy to deal really bad cuts